src/fios.h

```diff
@@ -131,7 +131,7 @@ class FileList {
 	 */
 	inline const FiosItem *Begin() const
 	{
-		return this->files.Begin();
+		return this->files.data();
```

Comment:
Make sure the vector contains files.
```suggestion
		assert(!this->files.empty());
		return this->files.data();
```